lib/tmp.js

```diff
@@ -349,10 +349,10 @@ function _prepareTmpFileRemoveCallback(name, fd, opts) {
       fs.closeSync(fdPath[0]);
     }
     catch (e) {
-      // under some node/windows related circumstances, a temporary file 
+      // under some node/windows related circumstances, a temporary file
       // may have not be created as expected or the file was already closed
       // by the user, in which case we will simply ignore the error
-      if (e.errno != -_c.EBADF && e.errno != -c.ENOENT) {
```

Comment:
@raszi, in my testing, the error codes had positive values. E.g. on Ubuntu, I consistently was getting an error because the temp file was already closed, and the value was EBADF (positive).  See https://github.com/kartotherian/tilerator/blob/master/lib/fileParser.js#L215